api/src/main/java/org/openmrs/api/OrderSetService.java

```diff
@@ -110,6 +110,8 @@
 	OrderSetMember getOrderSetMemberByUuid(String uuid);
 
 	/**
+	 * Get order set attribute by uuid from database
```

Comment:
I would drop `database`. That is a detail of the data access layer.

Reply:
i have removed it